packages/cdn/fastly.toml

```
@@ -0,0 +1,11 @@
+# This file describes a Fastly Compute package. To learn more visit:
+# https://developer.fastly.com/reference/fastly-toml/
+
+description = "Forma 36 CDN"
+language = "javascript"
+manifest_version = 2
+name = "@contentful/f36-cdn"
```

```suggestion
name = "f36-cdn"
```

I think with the `@` and `/` it might fail to successfully deploy.